FileCheckCA.sh

```diff
@@ -8,29 +8,23 @@
 # This example was used to check for leftover FileWave remnants
 #
 
-path1=/var/FileWave/
-path2=/usr/local/sbin/FileWave.app
-path3=/sbin/fwcontrol
-path4=/Applications/FileWave\ Kiosk.app
-path5=/System/Library/StartupItems/FWClient/
-
-
-if [ -e "$path1" ]; then echo "Yes: '$path1'"
-else echo "Not found: '$path1'"
-fi
-
-if [ -e "$path2" ]; then echo "Yes: '$path2'"
-else echo "Not found: '$path2'"
-fi
-
-if [ -e "$path3" ]; then echo "Yes: '$path3'"
-else echo "Not found: '$path3'"
-fi
-
-if [ -e "$path4" ]; then echo "Yes: '$path4'"
-else echo "Not found: '$path4'"
-fi
-
-if [ -e "$path5" ]; then echo "Yes: '$path5'"
-else echo "Not found: '$path5'"
-fi
+paths=(
+  "/var/FileWave/"
+  "/usr/local/sbin/FileWave.app"
+  "/sbin/fwcontrol"
+  "/Applications/FileWave Kiosk.app"
+  "/System/Library/StartupItems/FWClient/"
+)
+
+ret=0
```

Comment:
Appreciate the DRY logic, this particular script is producing file search output for an inventory system so we don't need/want it to produce an error code.